src/TulsiGenerator/PBXTargetGenerator.swift

```diff
@@ -569,6 +572,7 @@ final class PBXTargetGenerator: PBXTargetGeneratorProtocol {
         addFileReference(bridgingHeader)
       }
       let enableModules = (ruleEntry.attributes[.enable_modules] as? Int) == 1
+      let moduleName = ruleEntry.attributes[.module_name] as! String
```

Comment:
It would be better to keep `moduleName` optional for targets that should not have module names